src/matrix_comps.jl

```diff
@@ -634,3 +634,26 @@ function innovation_form(sys::ST; sysw=I, syse=I, R1=I, R2=I) where ST <: Abstra
 	K = kalman(sys, covar(sysw,R1), covar(syse, R2))
 	ST(sys.A, K, sys.C, Matrix{eltype(sys.A)}(I, sys.ny, sys.ny), sys.timeevol)
 end
+
+"""
+    predictor(sys::AbstractStateSpace, R1, R2)
+    predictor(sys::AbstractStateSpace, K)
+
+Return the predictor system
+x̂' = (A - KC)x̂ + Bu + Ky
```

Comment:
```suggestion
x̂' = (A - KC)x̂ + Bu + Ky
```

Note sure what's wrong with the hats. Not sure if this suggestion is better.